docs/hosting/custom_en-US.json

```diff
@@ -0,0 +1,182 @@
+{
+    "app": {
+        "name": "Parsec Cloud",
+        "developer": "Parsec Cloud",
+        "developerLink": "https://parsec.cloud/en",
+        "projectSources": "https://github.com/Scille/parsec-cloud",
+        "issueUrl": "https://github.com/Scille/parsec-cloud/issues/{issue}",
+        "licenseLink": "https://raw.githubusercontent.com/Scille/parsec-cloud/{version}/LICENSE",
+        "history": "https://docs.parsec.cloud/en/{version}/history.html",
+        "pricingLink": "https://parsec.cloud/en/pricing/"
+    },
+    "quit": {
+        "title": "Exit Parsec"
+    },
+    "globalErrors": {
+        "clientRevoked": "You have been revoked from this organization. You can still see some of the data you had access to, but most features of Parsec will not work.",
+        "clientFrozen": "You have been frozen in this organization. Most features of parsec will not work. Contact an administrator.",
+        "incompatibleEnvironment": "Your browser is not compatible with Parsec yet.",
+        "incompatibleWithSafari": "Parsec does not yet support Safari. Please use another browser such as Chrome, Firefox or Edge."
+    },
+    "link": {
+        "appIsBusy": "Parsec is busy. Please close any opened window before clicking on the link again."
+    },
+    "notification": {
+        "serverOnline": "You are connected to the Parsec server",
+        "serverOffline": "You are not connected to the Parsec server"
+    },
+    "HomePage": {
+        "sidebar": {
+            "tagline": "Keep control over your data confidentiality"
+        },
+        "profile": {
+            "settings": {
+                "description": "Manage your Parsec settings"
+            },
+            "account": {
+                "description": "Manage your Parsec account"
+            }
+        },
+        "topbar": {
+            "welcome": "Welcome to Parsec",
+            "updateConfirmTitle": "Update Parsec",
+            "updateYes": "Update Parsec"
+        },
+        "organizationList": {
+            "errors": {
+                "noStorage": "Unable to list or save your organizations: the application storage is not available. Make sure your disk is not running out of space and that you have read and write permissions to Parsec's configuration folder."
+            }
+        },
+        "noDevices": {
+            "titleCreateOrga": "New to Parsec?"
+        }
+    },
+    "CreateOrganization": {
+        "serverAddress": "Parsec server address",
+        "serverAddressPlaceholder": "parsec3://",
+        "useParsecServer": "Use the main Parsec server",
+        "useMyOwnServer": "I have my own Parsec server",
+        "saas": "Parsec SaaS",
+        "acceptTOS": {
+            "label": "By using Parsec, I accept the ",
+            "tosLink": "https://parsec.cloud/en/gtcs-parsec/",
+            "privacyPolicyLink": "https://parsec.cloud/en/politique-de-confidentialite/"
+        },
```

Comment:
Isn't this sent by the server? If I'm not mistaken, there is a `get_tos` route that exposes the actual TOS to be accepted.

:x:
```suggestion
```